fusionfusion/annotationFunction.py

```diff
@@ -222,23 +230,23 @@ def merge_fusion_result(input_dir, output_file_path):
         with open(input_dir + "/ms2.fusion.result.txt") as hIN:
             for line in hIN:
                 F = line.rstrip('\n').split('\t')
-                key = '\t'.join(F[0:7] + F[8:12])
+                key = '\t'.join(F[0:7] + F[8:])
```

Comment:
This change seems to affect the traditional case, but is it intended?

Reply:
I intentionally omitted those at the time, but after rechecking the code, the changes around here indeed look needless now 🤔 
Fortunately, they don’t seem to do any real harm unless we specify more than one of the three options `--star`, `--ms2` and `--th2` at the same time, but I’ll revert them either way.
Thank you for pointing this out!

Reply:
I've reverted these portions of the change at f05ea5b 👍  Also, I’ve reconfirmed that the result is identical to before.